### PR TITLE
Build ncbi-amrfinderplus for linux-aarch64

### PR DESCRIPTION
**Pull request by jmarshall**
Add additional-platforms entries.

----

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

Note that the <code>@BiocondaBot please merge</code> command is now depreciated. Please just squash and merge instead.

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by jmarshall** [comment]
Alas, many more prerequisites need to be looked at first…

**Comment by rpetit3** [comment]
started blast here: https://github.com/bioconda/bioconda-recipes/pull/46237

but like you said, dependencies all the way down

**Comment by jmarshall** [comment]
Separate hmmer off into PR #46288.

File: recipes/ncbi-amrfinderplus/build.sh [hunk 1]
```diff
@@ -13,6 +13,14 @@ CXXFLAGS="${CXXFLAGS} -O3 -D_LIBCPP_DISABLE_AVAILABILITY"
 # Get StxTyper source as well
 git submodule update --init
 
+case $(uname -m) in
+    aarch64 | arm64)
+        CXXFLAGS="${CXXFLAGS} -funsigned-char"
+        ;;
+    *)
+        ::
```
Comment (martin-g) [inline recipes/ncbi-amrfinderplus/build.sh:21]:
```suggestion
        ;;
```

**Comment by martin-g** [comment]
@BiocondaBot please fetch artifacts

**Comment by martin-g** [comment]
@BiocondaBot please add label

**Review by martin-g — APPROVED** [review]

**Pull request merged**